reward/rewardManager.go

```diff
@@ -34,3 +41,112 @@ type governanceHelper interface {
 func isEmptyAddress(addr common.Address) bool {
 	return addr == common.Address{}
 }
+
+type RewardManager struct {
```

Comment:
How about `RewardDistributor`?

Reply:
what about just Reward ?? How do you think about it?

Reply:
+1 for `RewardDistributor` instead of just `Reward`

Reply:
reflected